src/fpcore-visitor.rkt

```diff
@@ -67,11 +68,11 @@
      (vterm visitor ctx
             (visit-cast body)
             (visit-op_ 'cast (list body)))]
-    [(list (? operator? operator) args ...)
-     (vterm visitor ctx
+    [(list operator args ...) 
```

Comment:
This is because we need to be able to catch function calls somehow, without knowing in advance what they are called.

Do we need to move this rule down under the terminals, so we don't accidentally match terminals as ops with no arguments?

Reply:
I don't think so; terminals are never contained in a list. If FPCore doesn't support nullary functions, you could always change the match rule to `args ...+`.